swan-lake/development-tutorials/source-code-dependencies/package-references.md

```diff
@@ -269,6 +266,28 @@ graalvm = true        # Enable the GraalVM image generation
 graalvmBuildOptions = "--option1 --option2"        # Additional native-image options
 ```
 
+### Modules
+
+The [[package.modules]] array allows you to define metadata for individual modules. The following example demonstrates how to specify this information:
+
+``` toml
+[[package.modules]]
+name = "winery.foo"
+export = true
+readme = "README.md"
+
+[[package.modules]]
+name = "winery.bar"
+export = true
+readme = "ModuleReference.md"
+```
+
+The optional export field indicates whether the module is public. Only public modules will be visible to the outside. By default, only the default module is public.
+
+The optional readme field specifies the path to the module's documentation file in Markdown. If this field not provided, `README.md` is assumed to be the documentation file if it is available.
+
+Note that metadata for the default module is defined under the [package] table. Therefore, the name field in the [[package.modules]] array cannot match the default module name.
```

Comment:
Shall we add this as a note?

Reply:
Addressed with a61e7a2db16807cc17377b542957940bbae66d9c